README.md

```diff
@@ -93,63 +112,172 @@ const strapiConfig = {
       },
     },
   ],
+  // ...
 };
 ```
 
-#### Internationalization support
-
-Strapi now supports [internationalization](https://strapi.io/documentation/developer-docs/latest/development/plugins/i18n.html#installation). But by default, this plugin will only fetch data in the default locale of your Strapi app. If your content types are available in different locales, you can also pass an entity definition object to specify the locale you want to fetch for a content type. Use the `all` value to get all available locales on a collection type.
-
 #### Draft content
 
 Strapi now supports [Draft and publish](https://strapi.io/documentation/developer-docs/latest/concepts/draft-and-publish.html#draft-and-publish), which allows you to save your content as a draft and publish it later. By default, this plugin will only fetch the published content.
 
 But you may want to fetch unpublished content in Gatsby as well. To do so, find a content type that has draft & publish enabled, and add an entity definition object to your config. Then, use the query string option to specify the [publication state](https://strapi.io/documentation/developer-docs/latest/developer-resources/content-api/content-api.html#publication-state) API parameter.
 
-#### TODO
+**Path:** `./gatsby.config.js`
 
-## Querying data
+```javascript
+const strapiConfig = {
+  // ...
+  collectionTypes: [
+    {
+      singularName: 'article',
+      queryParams: {
+        publicationState: 'preview',
+        populate: {
+          category: { populate: '*' },
+          cover: '*',
+          blocks: {
+            populate: '*',
+          },
+        },
+      },
+    },
+  ],
+  // ...
+};
+```
 
-You can query Document nodes created from your Strapi API like the following:
+Then in your GraphQL query you should be able to display published content by using the following query:
+
+```graphql
+{
+  allStrapiArticle(filter: { publishedAt: { ne: null } }) {
+    nodes {
+      id
+    }
+  }
+}
+`
+```
+
+#### Image optimisation
+
+By default all medias are downloaded in the Gatsby file system.
+To query your asset use the following query:
 
 ```graphql
 {
   allStrapiArticle {
     nodes {
-      author {
-        name
-        avatar {
-          localFile {
-            childImageSharp {
-              gatsbyImageData
-            }
+      cover {
+        localFile {
+          childImageSharp {
+            gatsbyImageData(formats: NO_CHANGE)
```

Comment:
Is there any reason to use formats: NO_CHANGE here? In the starter I always used a plain gatsbyImageData